lib/rcfpch/rcf_pch_rpc.c

```diff
@@ -1844,9 +1844,9 @@ rcf_pch_rpc(struct rcf_comm_connection *conn, int sid,
     rpcs = rcf_pch_find_rpcserver(server);
     if (rpcs == NULL)
     {
-        ERROR("Failed to find RPC server %s", server);
+        RING("Failed to find RPC server %s", server);
```

Comment:
"Failed" sounds like an error and it is unclear why it is ignored, if I see in in the log.

Let's rephrase it as "RPC server '%s' not found".

If it is OK, I'd suggest to make it INFO, however other reviewed could disagree with me. @ol-arteman @ol-alexandra ?

Reply:
> If it is OK, I'd suggest to make it INFO

Is INFO included in the logs by default?

The usecases I personally have sometimes require to be able to see this log in case the RPC went down due to application restart etc.

Reply:
JFYI I'm still unhappy with an error message as I said above

Reply:
Fixed.